dlrover/python/common/diagnosis.py

```diff
@@ -74,3 +77,17 @@ def get_timestamp(self) -> int:
 
     def get_type(self) -> str:
         return DiagnosisDataType.CHIPMETRICES
+
+
+def should_relaunch_worker(log_file: str) -> bool:
+    if len(log_file) == 0:
+        return False
+    try:
+        with open(log_file, "r") as f:
+            error = "error code is 507035"
```

Comment:
These kind of error should be defined as constant in a independent class or module. There will certainly be more such error info in the future.